app/views/organizations/dashboard/index.html.erb

```diff
@@ -0,0 +1,104 @@
+ <!-- Sidebar -->
+ <nav class="navbar-vertical navbar">
+     <div class="vh-100" data-simplebar>
+         <!-- Brand logo -->
```

Comment:
I think this makes sense to be put in a partial on it's own and a layout for dashboard like pages.

Reply:
Ah yes now I know what we're doing.